http/http-api/src/main/java/com/okta/commons/http/RequestUtils.java

```diff
@@ -19,13 +19,19 @@
 import java.io.UnsupportedEncodingException;
 import java.net.URI;
 import java.net.URLEncoder;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
 import java.util.Locale;
 
 /**
  * @since 0.5.0
  */
 public final class RequestUtils {
 
+    private static final DateTimeFormatter DATE_TIME_FORMATTER =
+        DateTimeFormatter.ofPattern("EEE MMM d HH:mm:ss zzz yyyy", Locale.US);
```

Comment:
Don't hardcode a locale, you probably want something like:
```suggestion
        DateTimeFormatter.ISO_DATE_TIME.withZone(ZoneOffset.UTC);
```

Reply:
It doesn't work with this change.  Will need to check why.

Reply:
Updated, works now.